ng2ts/tests/templateAddActionTest.ts

```diff
@@ -26,12 +26,14 @@ describe('Blabla', () => {
 
   it(`app.component.ts: Add a 'search' method on the component, that takes a 'searchString' parameter.`, () => {
     const fixture = TestBed.createComponent(AppComponent);
+    const paramName = fixture.componentInstance.search.toString();
     fixture.detectChanges();
     chai.expect(fixture.componentInstance.search, `Search should be a function`).is.a('function');
     chai.expect(fixture.componentInstance.search.length, `Search should take one parameter`).equals(1);
+    chai.expect(paramName.toString().slice(paramName.indexOf('(') + 1, paramName.indexOf(')'))).equals('searchString');
```

Comment:
I think @prestonvanloon was suggesting to use toContain instead of equals. 

I'd suggest to put this one off and not submit last minute as string manipulations in tests are very unpredictable (my guess is that this particular implementation won't pass if a type is added. 

We should get together when there is more time and do some fun AST digging.